sources/packages/backend/apps/db-migrations/src/sql/Student/Update-gender-options.sql

```diff
@@ -0,0 +1,27 @@
+UPDATE
+  sims.students
+SET
+  gender = 'man'
+WHERE
+  gender = 'male';
+
+UPDATE
+  sims.students
+SET
+  gender = 'nonBinary'
+WHERE
+  gender = 'X';
+
+UPDATE
+  sims.students
+SET
+  gender = 'woman'
+WHERE
+  gender = 'female';
+
+UPDATE
+  sims.students
+SET
+  gender = NULL
+WHERE
+  gender = 'preferNotToAnswer';
```

Comment:
Since this option was not in DB, is there a reason to include it in the migration?

Reply:
I couldn't find examples of it, but I thought that non-answers were being saved as blanks from the ticket. What should those actually have been saved as previously?

Reply:
The DB conversion requested in the ticket is below unless I am missing something. Please let me know.

| Current    | Updated To |
| -------- | ------- |
| Male/male| Man/man    |
|Female/female | Woman/woman     |
| Unknown/unknown|  Prefer not to answer/preferNotToAnswer    |
|Diverse/diverse|Non-Binary/nonBinary|